packages/devextreme/js/__internal/ui/list/m_list.base.ts

```diff
@@ -345,6 +371,15 @@ export const ListBase = CollectionWidget.inherit({
   },
 
   _init() {
+    const { collapsibleGroups } = this.option();
+
+    if (collapsibleGroups) {
+      this._activeStateUnit = [
+        ...DEFAULT_ACTIVE_STATE_UNIT_ARRAY,
+        `.${LIST_GROUP_HEADER_CLASS}`,
+      ].join(',');
```

Comment:
[missed scenario?]
What if `collapsibleGroups` is changed at runtime?
I also see no tests for this case

Reply:
[10038c0b8e7df4df3c692d3c1f64327e9da1d4b0], [88dd969107513bc1a62715a7692d711a986024c7]